src/components/select/SingleSelectConnected.tsx

```diff
@@ -18,15 +22,19 @@ export interface ISingleSelectOwnProps extends ISelectProps {
     items?: IItemBoxProps[];
     buttonPrepend?: React.ReactNode;
     noFixedWidth?: true;
+    canClear?: true;

```

Comment:
With the new typing file working properly, tslint cries over using `true` as type in common repo. I think we should revert to using `boolean` and for `noFixedWidth` also.